logicmonitor/provider.go

```diff
@@ -35,19 +36,20 @@ func Provider() terraform.ResourceProvider {
 		},
 
 		DataSourcesMap: map[string]*schema.Resource{
-			"logicmonitor_device_group": dataSourceFindDeviceGroups(),
 			"logicmonitor_collectors":   dataSourceFindCollectors(),
+			"logicmonitor_device_group": dataSourceFindDeviceGroups(),
 		},
 		ConfigureFunc: providerConfigure,
 	}
 }
 
 func providerConfigure(d *schema.ResourceData) (interface{}, error) {
+	company := fmt.Sprintf("%s.logicmonitor.com", d.Get("company").(string))
```

Comment:
we should probably be a bit more defensive here and proactively determine whether or not to append .lm.com. I easily envision confused users setting "company" or "company.logicmonitor.com". We should be able to handle both.

Reply:
good point, i can either make a check or just always strip ".logicmonitor.com" from d.Get("company").(string)

Reply:
yeah either/or is fine

Reply:
done, will show up in next update